biopet-utils/pom.xml

```diff
@@ -31,6 +31,11 @@
 
     <dependencies>
         <dependency>
+            <groupId>com.typesafe.play</groupId>
```

Comment:
I think I will stay with play for now. I do agree that smaller liraries are nice but seeing the future plan I have the play/akka framework does suit better. I will not have 2 libraries inside biopet that overlap.